control-plane/agents/core/src/volume/specs.rs

```diff
@@ -587,6 +607,19 @@ impl ResourceSpecsLocked {
         Ok(volume)
     }
 
+    // Delete the NexusInfo key from the persistent store.
+    // If deletion fails we just log it and continue.
+    async fn delete_nexus_info(key: &NexusInfoKey, registry: &Registry) {
+        match registry.delete_kv(&key.key()).await {
+            Ok(_) => {
+                tracing::info!(nexus_info_key=%key,"Deleted NexusInfo entry from persistent store")
```

Comment:
IIRC We don't usually log successful operations (exception being the reconcilers), so maybe we shouldn't do it here either.

Reply:
Changed this to a trace message as it might be useful to still have it.